core/templates/dev/head/pages/exploration_player/PlayerTranscriptService.js

```diff
@@ -119,7 +146,7 @@ oppia.factory('playerTranscriptService', ['$log', function($log) {
       return this.getCard(transcript.length - 1);
     },
     getNumSubmitsForLastCard: function() {
-      return this.getLastCard().answerFeedbackPairs.length;
+      return this.getLastCard().inputResponsePairs.length;
```

Comment:
Take a look at what uses this -- is it looking for the number of answer requests + hint requests, or just the former?

If so you may need a more complex computation here, and also to introduce a 'type' field on each element in the transcript so that you can count the relevant ones. Or just maintain a counter in this service and not have the type field.